paddle/fluid/operators/pool_op.cc

```diff
@@ -180,6 +180,12 @@ void Pool2dOpMaker::Make() {
       "operator."
       "If global_pooling = true, paddings and ksize will be ignored.")
       .SetDefault({0, 0});
+  AddAttr<bool>(
```

Comment:
Also update "AddComment" so show the math when exclusive=False?

Reply:
Done for add math in AddComment.